apps/src/templates/instructions.jsx

```diff
@@ -0,0 +1,59 @@
+var msg = require('../locale');
+
+module.exports = React.createClass({
+
+  propTypes: {
+    puzzleTitle: React.PropTypes.string,
+    instructions: React.PropTypes.string,
+    instructions2: React.PropTypes.string,
+    renderedMarkdown: React.PropTypes.string,
+    authoredHints: React.PropTypes.array,
+    markdownClassicMargins: React.PropTypes.bool,
+    aniGifURL: React.PropTypes.string
+  },
+
+  render: function () {
+
+    // The logic here needs some serious cleanup
+    var body;
+    if (this.props.renderedMarkdown) {
+      body = (<div 
+        className={ this.props.markdownClassicMargins ? "instructions-markdown classic-margins" : "instructions-markdown" }
+        dangerouslySetInnerHTML={{ __html: this.props.renderedMarkdown }}
+      />);
+    } else {
+      body = [<p className='dialog-title'>{ this.props.puzzleTitle }</p>];
+      if (this.props.instructions) {
+        body.push(<p dangerouslySetInnerHTML={{ __html: this.props.instructions }}/>);
+      }
+      if (this.props.instructions2) {
+        body.push(<p className='instructions2' dangerouslySetInnerHTML={{ __html: this.props.instructions2 }}/>);
```

Comment:
Yes, these were for the optional second line of instructions, styled differently, in star wars.  We didn't use markdown instructions in that tutorial (except for a special case in the final level) because they couldn't be loc'ed.